analyzer-models/src/main/java/ffb/analyzer/models/espn/TeamRecordOverall.java

```diff
@@ -0,0 +1,17 @@
+package ffb.analyzer.models.espn;
+
+public class TeamRecordOverall extends TeamRecordBase {
+    private float pointsFor;
```

Comment:
How is deserialization/serialization with `float` working? I was getting errors when getting trying to deserialize numbers > 1 with a float :shrug. I'd prefer `float` since we don't need all the precision of `double`.

Reply:
Float (de)serialization has worked fine for me. Stepping through the TeamRecordTests I can see everything populating as expected. It even handles converting doubles to value, such as the "percentage" field in the TeamRecordBase class.

Where were you having issues with floats? I could take a look to see what could be different. Maybe open a separate comment on your pull requests calling it out.

Reply:
Some of the tests I wrote were failing on the double. I didn't look into them much, and don't remember which ones.

Made issue #17 to track using doubles. We can clean this up with #18.